Sample_v3/Samples/SwiftUISimpleChat/UserListView.swift

```diff
@@ -2,88 +2,92 @@
-        let isUserMuted = (
-            userList.controller.client.currentUserController().currentUser?.mutedUsers
-                .contains(where: { $0.id == user.id })
-        )!
-        return HStack {
-            Text(user.name ?? "Missing name")
-            Spacer()
-            /// Show `mute` icon if user is muted.
-            if isUserMuted {
-                Image(systemName: "speaker.slash.fill")
+        /// View with user name and mute icon if user is muted.
+        private func userView(for user: ChatUser) -> some View {
+            let isUserMuted = (
+                userList.controller.client.currentUserController().currentUser?.mutedUsers
+                    .contains(where: { $0.id == user.id })
+            )!
+            return HStack {
+                Text(user.name ?? "Missing name")
+                Spacer()
+                /// Show `mute` icon if user is muted.
+                if isUserMuted {
+                    Image(systemName: "speaker.slash.fill")
+                }
             }
         }
-    }
     
-    /// `ActionSheet` with actions that can be taken on the `userController`(`mute`, `unmute`)
-    /// Will appear on long pressing the user cell.
-    private func actionSheet(for user: ChatUser) -> ActionSheet {
-        let userController = userList.controller.client.userController(userId: user.id)
-        return ActionSheet(title: Text(user.name ?? "User name"), message: Text(""), buttons: [
-            .default(Text("Mute")) { userController.mute() },
-            .default(Text("Unmute")) { userController.unmute() },
-            .cancel()
-        ])
-    }
+        /// `ActionSheet` with actions that can be taken on the `userController`(`mute`, `unmute`)
+        /// Will appear on long pressing the user cell.
+        private func actionSheet(for user: ChatUser) -> ActionSheet {
+            let userController = userList.controller.client.userController(userId: user.id)
+            return ActionSheet(title: Text(user.name ?? "User name"), message: Text(""), buttons: [
+                .default(Text("Mute")) { userController.mute() },
+                .default(Text("Unmute")) { userController.unmute() },
+                .cancel()
+            ])
+        }
     
-    /// Pagination. Load next channels if last item is reached.
-    private func loadNextIfNecessary(encounteredIndex: Int) {
-        guard encounteredIndex == userList.users.count - 1 else { return }
-        userList.controller.loadNextUsers()
+        /// Pagination. Load next channels if last item is reached.
+        private func loadNextIfNecessary(encounteredIndex: Int) {
+            guard encounteredIndex == userList.users.count - 1 else { return }
+            userList.controller.loadNextUsers()
+        }
     }
-}
 
-/// Protocol conformance needed for ActionSheet presenting.
-extension ChatUser: Identifiable {}
+    /// Protocol conformance needed for ActionSheet presenting.
+    extension ChatUser: Identifiable {}
```

Comment:
I already commented somewhere that this should live inside the SDK. Someone must have ignored it 🙃

Reply:
Fixed in 63e1bca7835606a1113024bab02705576844fd34